emanifest-py/README.md

```diff
@@ -93,29 +93,35 @@ save manifests by uploading new .json and/or .zip files require a file path.
 ### Examples:
 
 ```python
-from emanifest import RcrainfoClient
+from emanifest import new_client
+
+rcrainfo = new_client('preprod')
+rcra_client.auth('YOUR_API_ID', 'YOUR_API_KEY')
 
-rcrainfo = RcrainfoClient('preprod')
-rcrainfo.GetSiteDetails('VATEST000001')
+rcrainfo.get_site_details('VATEST000001')
 ```
 
 Once you've confirmed this is the correct site, you might search for manifests in transit from that site:
 
 ```python
-from emanifest import RcrainfoClient
+from emanifest import new_client
 
-rcrainfo = RcrainfoClient('preprod')
```

Comment:
I'd like to show that `new_client` is just a constructor for the RcrainfoClient object. Right now 
```python
foo = new_client('preprod')
```
is functionally equivalent to 
```python 
foo = RcrainfoClient('preprod')
```

Reply:
I'm not sure what the better way is. Just regular object initialization or via the constructor.